### PR TITLE
Humans_oM: Add versioning for changes to CvalueAnalysis

### LINKED ISSUE #1221: Humans_oM: Versioning required on change to CValueSettings
#### Broken rules:
Versioning not in place for the changes to CValueSettings

### PR DESCRIPTION
**Pull request by rolyhudson**
### Issues addressed by this PR
Property name change versioned for CValueSettings
Previous version attribute for the CValueAnalysis method

Closes #1221 
Closes #https://github.com/BHoM/BHoM_Engine/issues

### Test files
Old components and old json strings in this [file](https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/BHoM/Humans_oM/%231221-CvalueVersioning/%231221_cvalueversioning.gh?csf=1&web=1&e=D0d4O0)

### CODE REVIEW

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

File: Humans_oM/Versioning_41.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "ToOld": {
+
+    }
+  },
+  "Property": {
+    "ToNew": {
+      "BH.oM.Humans.ViewQuality.CvalueSettings.RowTolerance": "BH.oM.Humans.ViewQuality.CvalueSettings.FarClippingPlaneDistance"
+    },
+    "ToOld": {
+      "BH.oM.Humans.ViewQuality.CvalueSettings.FarClippingPlaneDistance": "BH.oM.Structure.Elements.RigidLink.MasterNode.RowTolerance"
```
Comment (FraserGreenroyd) [inline Humans_oM/Versioning_41.json:22]:
```suggestion
      "BH.oM.Humans.ViewQuality.CvalueSettings.FarClippingPlaneDistance": "BH.oM.Humans.ViewQuality.CvalueSettings.RowTolerance"
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by rolyhudson** [comment]
@BHoMBot check versioning

**Review by FraserGreenroyd — APPROVED** [review]
Versioning errors reduced, this LGTM, thanks @rolyhudson :smile:

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by FraserGreenroyd** [comment]
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**